domains/transfer/TransferAgreementAutomation/Worker/Service/ProjectOriginWalletService.cs

```diff
@@ -36,9 +36,9 @@ AutomationCache cache
         this.cache = cache;
     }
 
-    public async Task TransferCertificates(TransferAgreementDto transferAgreement)
+    public async Task TransferCertificates(TransferAgreement transferAgreement)
     {
-        var header = SetupDummyAuthorizationHeader(transferAgreement.SenderId);
+        var header = SetupDummyAuthorizationHeader(transferAgreement.SenderId.ToString());
```

Comment:
I thought we removed the rest api? Why do we still need Auth headers?

Reply:
this is against the wallet